collectors/framework/framework_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/dcos/dcos-metrics/collectors"
 	mesosAgent "github.com/dcos/dcos-metrics/collectors/mesos/agent"
 	"github.com/dcos/dcos-metrics/producers"
-	"github.com/dcos/dcos-metrics/schema/metrics_schema"
+	metricsSchema "github.com/dcos/dcos-metrics/schema/metrics_schema"
```

Comment:
why only this import is changed but not the code?

Reply:
goimports was complaining about some unaliased imports; see https://github.com/dcos/dcos-metrics/pull/168/commits/7dea6ee57735317052ce91650982b80f6e286b00